de-lib/src/main/java/org/iplantc/de/admin/apps/client/views/grid/AdminAppsGridImpl.java

```diff
@@ -125,10 +127,19 @@ public void onSelectionChanged(SelectionChangedEvent<App> event) {
     }
 
     @Override
-    protected void onEnsureDebugId(String baseID) {
+    protected void onEnsureDebugId(final String baseID) {
         super.onEnsureDebugId(baseID);
 
+        acm.ensureDebugId(baseID);
         grid.asWidget().ensureDebugId(baseID + Belphegor.AppIds.GRID);
+        grid.addViewReadyHandler(new ViewReadyEvent.ViewReadyHandler() {
```

Comment:
I did this for all the views in Belphegor.  I'm not sure if `onEnsureDebugId` is the best place to put it since we're adding a handler to the grid, but it's only for setting the debug IDs, so I put it here for now.